core/src/apps/base.py

```diff
@@ -265,4 +270,4 @@ def boot() -> None:
 
     wire.experimental_enabled = storage.device.get_experimental_features()
 
-    workflow.idle_timer.set(storage.device.get_autolock_delay_ms(), lock_device)
+    workflow.idle_timer.set(storage.device.get_autolock_delay_ms(), lock_device_if_unlocked)
```

Comment:
Perhaps we should call `reload_settings_from_storage` here?

Reply:
Tiny bit of code duplication (which I missed the first time), we do the same two things there (plus setting display orientation which is idempotent). Dunno, probably not worth it for 2 lines.

Reply:
oh I see now. yes, that would make sense, but we would have to move `reload_settings_from_storage` to `common` probably?

Reply:
Moved it to `apps/base.py` in aa07da9dc7544056a73f89b586f9f797e5dabc79. Not sure where in `common` would it go, and some other apps import from `apps.base` already.